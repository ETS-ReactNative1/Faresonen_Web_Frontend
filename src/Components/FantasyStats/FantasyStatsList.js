import React from "react";
import { Header, List } from "semantic-ui-react";
import PropTypes from "prop-types";

const FantasyStatsList = props => (
  <div>
    <Header as="h2">{props.header}</Header>
    <List items={props.players} />
  </div>
);

FantasyStatsList.propTypes = {
  players: PropTypes.arrayOf(PropTypes.string).isRequired,
  header: PropTypes.string.isRequired
};

export default FantasyStatsList;
