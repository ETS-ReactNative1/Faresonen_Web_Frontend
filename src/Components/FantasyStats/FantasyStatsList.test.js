import React from "react";
import renderer from "react-test-renderer";
import FantasyStatsList from "./FantasyStatsList";

describe("<FantasyStatsList />", () => {
  it("Should render properly", () => {
    const tree = renderer.create(
      <FantasyStatsList
        players={[
          "£5,6 mill - Harry Kane",
          "£4,2 mill - Mohamed Salah",
          "£3,3 mill - Alexis Sanchez",
          "£2,9 mill - Romelu Lukaku"
        ]}
        header="Dyreste spillere"
      />
    );
    expect(tree).toMatchSnapshot();
  });
});
