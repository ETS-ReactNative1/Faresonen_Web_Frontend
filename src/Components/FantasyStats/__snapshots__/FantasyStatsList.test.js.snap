// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FantasyStatsList /> Should render properly 1`] = `
<div>
  <h2
    className="ui header"
  >
    Dyreste spillere
  </h2>
  <div
    className="ui list"
    role="list"
  >
    <div
      className="item"
      onClick={[Function]}
      role="listitem"
    >
      £5,6 mill - Harry Kane
    </div>
    <div
      className="item"
      onClick={[Function]}
      role="listitem"
    >
      £4,2 mill - Mohamed Salah
    </div>
    <div
      className="item"
      onClick={[Function]}
      role="listitem"
    >
      £3,3 mill - Alexis Sanchez
    </div>
    <div
      className="item"
      onClick={[Function]}
      role="listitem"
    >
      £2,9 mill - Romelu Lukaku
    </div>
  </div>
</div>
`;
